eng/tools/typespec-validation/src/index.ts

```diff
@@ -18,11 +18,27 @@ export async function main() {
       type: "string",
       short: "f",
     },
+    validateOldStructure: {
```

Comment:
I think whether a particular spec is using the "old" or "new" structure will need to be configuration in the repo itself.  For example, TSV could default to requiring the new structure, but use the suppressions mechanism to "configure" the FolderStructure rule.  Something like:

```
suppressions.yaml
- tool: TypeSpecValidation
  rule: FolderStructure
  sub-rule: FolderStructureV2
  reason: Not ready to migrate to TypeSpec folder structure v2
```

The rule would then treat "suppress v2" as "enforce v1".

Reply:
Yep, we could do it this way and I think it makes sense. Does this need more discussion with other people in order for us to decide to go this route, or can we assume "this is the way"?